core/src/cg/group4/util/audio/AudioPlayer.java

```diff
@@ -47,7 +66,9 @@ public AudioPlayer() {
         cPreferences = Gdx.app.getPreferences("AUDIO");
         cAudioEnabled = cPreferences.getBoolean("ENABLED", true);
         cDefaultMusic = Gdx.audio.newMusic(Gdx.files.internal("music/Summer Day.mp3"));
```

Comment:
Why are you using 
```java 
Gdx.audio.newMusic(Gdx.files.internal("......."));
```

You should be using the Assets class to load your Music files (asynchronously).

Just like you use the Assets in the 3 lines below (be consistent).

```java
cEventStartedSound = Assets.getInstance().getSound("sounds/eventOccured.mp3");
cTaskCompletedSound = Assets.getInstance().getSound("sounds/completedTask.wav");
cEventCompletedSound = Assets.getInstance().getSound("sounds/eventCompleted.mp3");
```